pro/src/new_components/StockThingEventForm/__specs__/validationSchema.spec.tsx

```diff
@@ -0,0 +1,217 @@
+import '@testing-library/jest-dom'
+
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { Formik } from 'formik'
+import React from 'react'
+
+import { IStockThingFormProps } from 'new_components/StockThingForm/StockThingForm'
+import { SubmitButton } from 'ui-kit'
+import { getToday } from 'utils/date'
+
+import { STOCK_THING_EVENT_FORM_DEFAULT_VALUES } from '../constants'
+import StockThingEventForm from '../StockThingEventForm'
+import { getValidationSchema } from '../validationSchema'
+
+jest.mock('utils/date', () => ({
+  ...jest.requireActual('utils/date'),
+  getToday: jest
+    .fn()
+    .mockImplementation(() => new Date('2022-12-25T00:00:00Z')),
+}))
+
+const today = getToday()
+const yesterday = getToday()
+const tomorrow = getToday()
+yesterday.setDate(yesterday.getDate() - 1)
+tomorrow.setDate(tomorrow.getDate() + 1)
+
+const renderStockThingEventForm = ({
+  minQuantity,
+}: {
+  minQuantity?: number | null
+} = {}) => {
+  const props: IStockThingFormProps = { today }
+  return render(
+    <Formik
+      initialValues={STOCK_THING_EVENT_FORM_DEFAULT_VALUES}
+      onSubmit={() => {}}
+      validationSchema={getValidationSchema(minQuantity)}
+    >
+      {({ handleSubmit }) => (
+        <form onSubmit={handleSubmit}>
+          <StockThingEventForm {...props} />
+          <SubmitButton>Submit</SubmitButton>
+        </form>
+      )}
+    </Formik>
+  )
+}
+
+describe('StockThingEventForm:validationSchema', () => {
+  it('test required fields', async () => {
+    renderStockThingEventForm()
+
+    await userEvent.click(screen.getByRole('button', { name: 'Submit' }))
+    expect(screen.getByTestId('error-eventDatetime')).toBeInTheDocument()
+    expect(screen.getByTestId('error-eventTime')).toBeInTheDocument()
+    expect(screen.getByTestId('error-price')).toBeInTheDocument()
+    expect(
+      screen.queryByTestId('error-bookingLimitDatetime')
+    ).not.toBeInTheDocument()
+    expect(screen.queryByTestId('error-quantity')).not.toBeInTheDocument()
+  })
+
+  const dataSetEventDateTime: Array<number> = [
+    today.getDate(),
+    tomorrow.getDate(),
+  ]
+  it.each(dataSetEventDateTime)(
+    'should not display eventDatetime error',
+    async eventDateTime => {
+      renderStockThingEventForm()
+
+      await userEvent.click(screen.getAllByPlaceholderText('JJ/MM/AAAA')[0])
```

Comment:
Je trouve ca plus jolie de prendre 

```suggestion
      await userEvent.click(screen.getByLabelText('Date'))
```